azurerm/data_source_scheduler_job_collection.go

```diff
@@ -102,7 +102,7 @@ func dataSourceArmSchedulerJobCollectionRead(d *schema.ResourceData, meta interf
 		}
 		d.Set("state", string(properties.State))
 
-		if err := d.Set("quota", flattenAzureArmSchedulerJobCollectionQuota(properties.Quota)); err != nil {
+		if err := d.Set("quota", flattenAzureArmSchedulerJobCollectionQuota(properties.Quota, d.Get("quota").([]interface{}))); err != nil {
```

Comment:
in general we shouldn't access properties via `d.Get` in the Read method - since they can be absent when importing